.github/workflows/nlu-hyperopt.yml

```diff
@@ -0,0 +1,17 @@
+
+name: main
+on: [pull_request]
+
+jobs:
+  nlu-hyperopt:
+    name: NLU hyperoptimization
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: RasaHQ/nlu-hyperopt@rasa-1-compat
```

Comment:
Interesting. This will will run with only one worker then (whereas the master is the worker at the same time) 💡